src/stravavis/plot_elevations.py

```diff
@@ -21,9 +21,14 @@ def plot_elevations(df, output_file = 'elevations.png'):
         sharey = True,
         )
 
+    def fplt(x, y, **kwargs):
+        if y.max() - y.min() > 4000 or y.min() < -1000 or y.max() > 9000:
```

Comment:
Where do these 4000, -1000 and 9000 limits come from?

Reply:
I suppose very few people can climb 4000 meters in a ride. And there is no land above 9000m altitude on earth.